src/main/java/frc/robot/RobotContainer.java

```diff
@@ -39,44 +44,83 @@ public RobotContainer() {
     }
 
     private void configureBindings() {
```

Comment:
In this function, it would be better to do `// spotless:off` at the very beginning and `// spotless:on` at the end. This way the formatting doesn't get turned off for the whole rest of the file